inbox/headless/get-started.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: 'Headless Inbox'
+sidebarTitle: 'Get Started'
+description: 'A lightweight, standalone package for building custom in-app notification interfaces with Novu, providing essential functionalities and flexibility.'
+---
+
+The headless version of Novu’s notification library package provides users with a lightweight solution for integrating notification functionality into their web applications. With just the essential API methods, users can easily incorporate our notification system into any framework or vanilla JavaScript project, without being constrained by our default UI or dependencies.
+
+## Get Started
+
+<Steps>
+<Step title="Installation">
+```bash
+npm install @novu/js
+```
+</Step>
+<Step title="Import Package">
+```typescript
+import { Novu } from "@novu/js";
+```
+</Step>
+<Step title="Initialize Session">
+<Tabs>
+<Tab title="US" >
+```typescript
+import { Novu } from "@novu/js";
+
+const novu = new Novu({
+  subscriberId: "SUBSCRIBER_ID",
+  applicationIdentifier: "APPLICATION_IDENTIFIER",
+});
+```
+</Tab>
+<Tab title="EU">
+```typescript
+import { Novu } from "@novu/js";
+
+const novu = new Novu({
+  subscriberId: "SUBSCRIBER_ID",
+  applicationIdentifier: "APPLICATION_IDENTIFIER",
+  backendUrl: "https://eu.api.novu.co",
+  socketUrl: "https://eu.ws.novu.co",
+});
+```
+</Tab>
+<Tab title="HMAC Encryption">
+Read more about [HMAC Encryption](/react/advanced-configuration#hmac-encryption).
+```typescript
+import { Novu } from "@novu/js";
+
+const novu = new Novu({
+  subscriberId: "SUBSCRIBER_ID",
+  applicationIdentifier: "APPLICATION_IDENTIFIER",
+  subscriberHash: "SUBSCRIBER_HASH_HMAC_ENCRYPTIon",
+});
+```
+</Tab>
+</Tabs>
+</Step>
+<Step title="Fetch Notifications">
+```typescript
+const notifications = await novu.notifications.list({
+  limit: 30,
+  read: false,
+  archived: false,
+});
+```
+</Step>
+<Step>
+Display `notifications` in your UI.
+</Step>
+</Steps>
+
+## Realtime Notifications
```

Comment:
I'm not sure if this section should be on the `Get Started` page, wdyt @SokratisVidros ?